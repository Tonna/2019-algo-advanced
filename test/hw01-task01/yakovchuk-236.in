30
1 1 2 2 1 3 5 2 4 9 2 9 11 4 12 11 11 14 4 6 6 4 16 3 12 1 1 3 22
6
3 4
23 2
14 5
12 6
15 4
29 1
