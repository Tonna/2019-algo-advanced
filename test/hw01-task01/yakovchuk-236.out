7
8
24
10
12
3
